tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -806,10 +806,12 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
               this.scopeSubtree(iframe);
               container.appendChild(iframe);
               const subdocument = iframe.contentDocument;
-              const baseUrl = tf_backend.getRouter()
+              const pluginBasePath = tf_backend.getRouter()
                   .pluginRoute(selectedDashboard, '/')
               const base = subdocument.createElement('base');
-              base.href = baseUrl;
+              base.setAttribute(
```

Comment:
Can you add a comment explaining why the indirection is necessary?

Reply:
Done. Added few lines.